owncloudApp/src/main/java/com/owncloud/android/ui/preview/PreviewImageActivity.kt

```diff
@@ -124,9 +126,18 @@ class PreviewImageActivity : FileActivity(),
     }
 
     private fun startObservingFileOperations() {
-        fileOperationsViewModel.removeFileLiveData.observe(this, Event.EventObserver {
-            if (it.isSuccess) {
-                finish()
+        fileOperationsViewModel.removeFileLiveData.observe(this, Event.EventObserver { uiResult ->
+            when (uiResult) {
+                is UIResult.Error -> {
+                    dismissLoadingDialog()
+                    showErrorInSnackbar(R.string.remove_fail_msg, uiResult.getThrowableOrNull())
+                }
+
+                is UIResult.Loading -> showLoadingDialog(R.string.wait_a_moment)
+                is UIResult.Success -> {
+                    dismissLoadingDialog()
+                    finish()
+                }
```

Comment:
Ok!! Thanks for this improvement in image and video previews, so that we can keep a consistent behaviour with other previews 👍